keps/sig-api-machinery/4020-unknown-version-interoperability-proxy/README.md

```diff
@@ -136,47 +135,49 @@ incorrectly or objects being garbage collected mistakenly.
 
 ## Proposal
 
-We will use the existing `StorageVersion` API to figure out which group, versions,
-and resources an apiserver can serve.
+We will use the existing [Aggregated Discovery](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/3352-aggregated-discovery/README.md) mechanism to fetch which group, versions and resources an apiserver can serve.
```

Comment:
nit for the entire document: auto format so the lines are cut off at a certain character limit.

Reply:
Done